package/skelcd-control-MicroOS.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Jan 24 21:12:31 UTC 2022 - David Diaz <dgonzalez@suse.com>
+
+- Add bootloader pattern for all roles
+  (gh#yast/skelcd-control-MicroOS/#47).
```

Comment:
I would mention alberto here something like `(thanks to aplanas)` to give him credit for contribution :wink:

Reply:
Right! Excuse me, I thought I have writing his name and mail there! Thanks!